guillotina/request.py

```diff
@@ -510,9 +510,15 @@ def content(self):
 
     @reify
     def content_type(self):
-        """Return raw payload stream."""
         return self.headers.get("content-type")
 
+    @reify
+    def content_length(self) -> Optional[int]:
+        try:
+            return int(self.headers["content-length"])
+        except KeyError:
+            return None
```

Comment:
Should it be 0?

Reply:
and if the users sends a string with alpha chars instead of numbers, we want to fail or return `None/0` ?

Reply:
I just looked at what we had before with aiohttp https://github.com/aio-libs/aiohttp/blob/50753eace15da24a9fb9774bcfcaa1dc7650164e/aiohttp/helpers.py#L662

In this case, if user sends alpha chars, it would fail, which I think is ok no?

Reply:
Ok, yes!